css/main.css

```diff
@@ -259,90 +141,80 @@ input {
 	text-shadow: 3px 3px 6px black;
 }
 
-.review1 {
-	background-color: rgba(122, 242, 132, 0.461);
-	font-family: Verdana, Geneva, Tahoma, sans-serif;
-	border-radius: 6px;
+/* || Index Page */
+
+.hoursTable {
```

Comment:
.hoursTable is not assigned to the Hours div on index.php
color assignment to Hours of Operation is missing, should be green not black text.

Reply:
fixed!